packages/babel-traverse/src/scope/lib/renamer.ts

```diff
@@ -26,6 +27,22 @@ const renameVisitor: Visitor<Renamer> = {
       }
     }
   },
+  ObjectProperty(path: NodePath<ObjectProperty>) {
+    const { extra, key, value, computed, shorthand } = path.node;
+
+    if (computed) {
+      return;
+    }
+
+    if (!shorthand) {
+      return;
+    }
```

Comment:
When `shorthand===true`, must `computed===false`? I'm not entirely sure about this, any other reviewers know? Thanks!

Reply:
Uh yes, shorthand properties cannot be computed.